cnv/cnv_users_guide/cnv-automating-management-tasks.adoc

```diff
@@ -0,0 +1,19 @@
+[id="cnv-automating-management-tasks"]
+= Automating management tasks
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-automating-management-tasks
+toc::[]
+
+You can automate {ProductName} management tasks by using Red Hat Ansible
+Automation. Learn the basics by using an Ansible Playbook to create a new
+virtual machine.
+
+include::modules/cnv-about-red-hat-ansible-automation.adoc[leveloffset=+1]
+
+include::modules/cnv-automating-virtual-machine-creation-with-ansible.adoc[leveloffset=+1]
+
+To verify that the virtual machine was created, try to
+xref:../../cnv/cnv_users_guide/cnv-accessing-vm-consoles.adoc#cnv-accessing-vm-consoles[access the VM console].
```

Comment:
It seems that the `#cnv-accessing-vm-consoles` anchor doesn't exist on this page. Did you mean `#cnv-accessing-vm-consoles-cli` ?

Reply:
@huffmanca I actually meant to link to the top of the page (the assembly :context:). Is that okay as-is? If not, I'm happy to change it.